src/main/java/org/broadinstitute/hellbender/tools/gvs/common/PloidyUtils.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.tools.gvs.common;
+
+import htsjdk.samtools.util.CollectionUtil;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.Lazy;
+import htsjdk.variant.variantcontext.VariantContext;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class PloidyUtils {
+    /**
+     * This copies the PAR definition used in the tool FindMendelianViolations with the addition of the Y regions
+     * https://github.com/broadinstitute/picard/blob/master/src/main/java/picard/vcf/MendelianViolations/FindMendelianViolations.java#L203
+     * Wikipedia data: https://en.wikipedia.org/wiki/Pseudoautosomal_region
+     */
+    private static final Set<String> PSEUDO_AUTOSOMAL_REGIONS_DEFINITION = CollectionUtil .makeSet("X:60001-2699520", "X:154931044-155260560", "chrX:10001-2781479", "chrX:155701383-156030895", "Y:10001-2649520", "Y:59034050-59363566", "chrY:10001-2781479", "chrY:56887903-57217415");
```

Comment:
I'm a little surprised this doesn't exist already somewhere in GATK

Reply:
So was I, to be honest.  I grepped a bunch of terms related to PARs, and the best I could find is that FindMendelianViolations tool that I linked in the comments.  Strangely, that one doesn't even include the regions on the y chromosome, for whatever reason. AND its definition and implementation was private, so I couldn't even just import it here.

It's possible I missed something, but at the very least I can say that anything that may exist is pretty non-obvious.

Reply:
ask in the methods channel perhaps?

Reply:
Maybe it would be defined in the reference fasta itself? I've seen some with XY or PAR as the contig

Reply:
Asked in methods, and they don't really have an solution to our use case